server/src/internalClusterTest/java/org/opensearch/indices/store/IndicesStoreIntegrationIT.java

```diff
@@ -168,6 +168,97 @@ public void testIndexCleanup() throws Exception {
 
     }
 
+    /**
+     * Verify files are removed even if there is another relocation is in progress.
```

Comment:
Minor typo: another relocation in progress
or better
are removed even when another relocation is in progress

Reply:
Updated.